src/constants.js

```diff
@@ -149,6 +149,7 @@ export const PERSISTED_STATE_KEYS = [
   'recordSingleDayProfit',
   'revenue',
   'showNotifications',
+  'shovelUnlocked',
```

Comment:
Note that if we remove the new `shovelUnlocked` state property, we can remove this too!